### PR TITLE
CLOUD-12 Removed cruft from Payara Micro

### PR DESCRIPTION
**Pull request by Cousjava**
Jars no longer included: glassfish-embedded-shell,
glassfish-embedded-static-shell, jline-terminal-jna, jna, deployment-client, appserver-cli,domain backup and schema2beans.

# Testing

### Testing Performed
Manual restart of server and micro.
Also comparison of jars included in 5.194 and in this PR.

### Test Suites Run
- Java EE 8 Samples
- Microprofile Metrics
- Microprofile Health

### CODE REVIEW

**Comment by Cousjava** [comment]
Jenkins test please

File: appserver/packager/appserver-core/pom.xml [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+        <dependency>
+            <groupId>fish.payara.server.internal.admin</groupId>
+            <artifactId>appserver-cli</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>net.java.dev.jna</groupId>
+            <artifactId>jna</artifactId>
+            <version>${jna.version}</version>
+        </dependency>
```
Comment (pdudits) [inline appserver/packager/appserver-core/pom.xml]: jna is prerequisite for jline

File: appserver/packager/appserver-core/pom.xml [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+            <groupId>fish.payara.server.internal.extras</groupId>
+            <artifactId>appserv-rt-frag</artifactId>
+            <version>${project.version}</version>
+            <type>zip</type>
+        </dependency>
+        <dependency>
+            <groupId>fish.payara.server.internal.admin</groupId>
+            <artifactId>backup</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```
Comment (pdudits) [inline appserver/packager/appserver-core/pom.xml]: support for domain backup

**Comment by pdudits** [comment]
Since Jenkins don't exercise micro properly, can you yet run i. e. javaee8-samples and some Microprofile TCKs against it?

**Comment by Cousjava** [comment]
Jenkins test please

**Pull request merged**